app/models/response.rb

```diff
@@ -6,10 +6,43 @@ class Response < ApplicationRecord
 
   belongs_to :response_map, class_name: 'ResponseMap', foreign_key: 'map_id', inverse_of: false
   has_many :scores, class_name: 'Answer', foreign_key: 'response_id', dependent: :destroy, inverse_of: false
+  accepts_nested_attributes_for :scores
 
   alias map response_map
   delegate :questionnaire, :reviewee, :reviewer, to: :map
 
+  # response type to label mapping
+  KIND_LABELS = {
```

Comment:
Isn't this defined by a table somewhere?  Seems like this is a redundant definition.